core/src/main/java/io/kestra/core/models/tasks/ExecutionUpdatingTask.java

```diff
@@ -0,0 +1,11 @@
+package io.kestra.core.models.tasks;
+
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.runners.RunContext;
+
+/**
+ * Interface for tasks that modify the execution at runtime.
+ */
+public interface ExecutionUpdatingTask {
```

Comment:
All other task types are suffixed by _able_, maybe call it ExecutionUpdatableTask ?